src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/paragraph/index.js

```diff
@@ -0,0 +1,23 @@
+import loadable from '@loadable/component';
+
+const richTextTransforms = loadable(() =>
+  import(
+    /* webpackChunkName: 'rich-text-transforms' */ '@bbc/psammead-rich-text-transforms'
+  ),
+);
+
+const convertParagraph = async block => {
+  const {
+    candyXmlToRichText,
+    stringToRichText,
+  } = await richTextTransforms.load();
```

Comment:
Most of this is going to be duplicated for every block - seems there would be a better way to do this?  (totally fine with that being deferred to later though)

Reply:
Yeah most of this will need to be generalised and reused, im happy with doing this when needed too though